asteriks_diamond_loop.c

```diff
@@ -0,0 +1,23 @@
+/*Ava N.*/
+/*This has half a diamond on request of the user's preference of length*/
```

Comment:
Hi Ava, your code neither compiled nor ran. When I fixed your compilation error and ran it, it still didn't properly run. Your code's logic is very off - you should see Ms. Hsu or me about this.